### DISCUSSION TITLE
[LOL] $150 seems both cheap and expensive

### DISCUSSION THREAD

**Discussion by deckerst** [deckerst/aves#266] [General]
![Screenshot_20220603-223717_Firefox](https://user-images.githubusercontent.com/13898333/171866340-f1b5f1cf-4d46-4767-accd-75f1aa0ba528.jpg)

**Comment by kha84** [comment]
It's very sad actually what they do. They take some great opensource app, then they strip it - like changing its name, removing identity, change colors and icons. And then they're trying to sell on PlayMarket under different name. I've seen this kind of s.t was happening to other opensource apps and I was always reporting to Google about that as violations of licensing. When I knew that original peoduct was distributed under GPLv3 and these smartaleks were trying to make a proprietary thing out of it

**Reply by deckerst** [reply]
For me the saddest part is the fact that some developers are so desperate that they will submit proposals to work on this rip-off, throwing away ethics for a little money.

**Comment by kha84** [comment]
Well it's not the first time this happens and not the last. Remember a zillion clones of 2048 game or flappybird? It didn't start even then, it started many centuries ago. There were always some fraction of ppl who were able to create something new, solid and precious. And thousands of ppl living next door ready to rip it off. Don't be upset much about it. When you see more attempts to clone your creature it just means it gains more and more popularity as it should. Because it's literally a gem

**Comment by Poussinou** [comment]
Hi @deckerst 

Somebody just found this copycat of Aves on Google Play https://play.google.com/store/apps/details?id=deckers.thibault.aves.libre from [this developer](https://play.google.com/store/apps/developer?id=Schnupfeld).

I'm not familiair with BSD-3-Clause license but as far as I can see, this copycat doesn't respect this license. You can follow the procedure [here](https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1399222536) if you want to send a DMCA takedown request to Google Play.

**Reply by BenVB1** [reply]
Just flagged it as inappropriate on there. Hope others do the same.

**Reply by IzzySoft** [reply]
That copycat uploaded a lot of FOSS apps to Play which (for good reasons) have not been uploaded there by their original authors (as aras I could see it seems that **all** of the apps listed under that account are copycats). But as usual, those APKs were "enriched" by some monetization libs. Some authors [got copies of their apps down](https://fosstodon.org/@AAMfP/110040759238409645) from that list via DMCA.
